Vagrantfile

```diff
@@ -84,6 +84,8 @@ Vagrant.configure(2) do |config|
   # that one) as well as making sure all non-primary ones have "autostart: false" set
   # so that when we do "vagrant up", it doesn't spin up those machines.
 
+  config.vm.boot_timeout = 600
```

Comment:
I think it would be useful to leave a comment saying this is in seconds and its the time to wait till the VM is up and reachable